src/kernel/src/main.rs

```diff
@@ -89,6 +91,12 @@ fn main() -> ExitCode {
         }
     }
 
+    // Begin Discord Rich Presence after successful basic init.
+    let game_display = args.game.display().to_string();
```

Comment:
`Path::display` is a lossy conversion. It was designed for printing the path only.